Helm chart images: conda removed -> pip only, usage disclaimer added, minimized Dockerfile complexity (#533)
dask/dask-gateway: docs/source/install-kube.rst

should work fine. You may install any additional libraries or dependencies you
require.

We encourage you to maintain your own image for scheduler and worker pods as
this project only provides a `minimal image`_ for testing purposes which can be

> used as inspiration, but isn't recommended to be used as a base image.

Reviewer: ```suggestion
used as inspiration, but isn't recommended to be used in production.
```

Reviewer 2: It can be used in production, and I wouldn't even say we don't recommend using it as a base image, it's just that we won't do anything extra to support using it as a base image (adding extra build args, etc...).

> We encourage you to maintain your own image for scheduler and worker pods as
> this project only provides a `minimal image`_ for testing purposes which can be
> used as inspiration, but isn't recommended to be used as a base image.

Author: ```suggestion
We encourage you to maintain your own image for scheduler and worker pods as
this project only provides a `minimal image`_ for testing purposes.
```

I made this suggestion based on the discussion just above in https://github.com/dask/dask-gateway/pull/533/files#r852284873.

Reviewer: ok
